dune-project

```diff
@@ -18,11 +18,6 @@
  (synopsis "Bindings for Push Notification")
  (description "Bindings for Push Notification")
  (depends
-  (ocaml
-   (>= 4.05))
-  (dune
-   (>= 2.0))
-  (js_of_ocaml
-   (>= 3.6))
-  (js_of_ocaml-ppx
-   (>= 3.6))))
+  (ocaml (>= 4.05))
+  (dune (>= 2.0))
+  ezjs_min))
```

Comment:
That looks wrong to me. E.g. in the dune file you got: `preprocess (pps js_of_ocaml-ppx))`.

Reply:
ezjs_min will always imply js_of_ocaml-ppx/js_of_ocaml dependencies (as js_of_ocaml-ppx will always imply js_of_ocaml)